static/src/js/schemas/uiSchemas/collections/spatialInformation.jsx

```diff
@@ -0,0 +1,1375 @@
+/* eslint-disable react/react-in-jsx-scope */
+import React from 'react'
+import CustomSelectWidget from '../../../components/CustomSelectWidget/CustomSelectWidget'
+import '../../../components/PreviewMapTemplate/PreviewMapTemplate.scss'
+import PreviewMapTemplate from '../../../components/PreviewMapTemplate/PreviewMapTemplate'
+
+const spatialInformationUiSchema = {
+  'ui:submitButtonOptions': {
+    norender: true
+  },
+  'ui:heading-level': 'h3',
+  'ui:field': 'layout',
+  'ui:layout_grid': {
+    'ui:row': [
+      {
+        'ui:group': 'Spatial Information',
+        'ui:group-description': true,
+        'ui:col': {
+          md: 12,
+          children: [
+            {
+              'ui:row': [
+                {
+                  'ui:col': {
+                    md: 12,
+                    children: ['SpatialExtent']
+                  }
+                }
+              ]
+            },
+            {
+              'ui:row': [
+                {
+                  'ui:col': {
+                    md: 12,
+                    children: ['TilingIdentificationSystems']
+                  }
+                }
+              ]
+            },
+            {
+              'ui:row': [
+                {
+                  'ui:col': {
+                    md: 12,
+                    children: ['SpatialInformation']
+                  }
+                }
+              ]
+            },
+            {
+              'ui:row': [
+                {
+                  'ui:col': {
+                    md: 12,
+                    children: ['LocationKeywords']
+                  }
+                }
+              ]
+            }
+          ]
+        }
+      }
+    ]
+  },
+  SpatialExtent: {
+    'ui:heading-level': 'h4',
```

Comment:
In general, the headings on this page don't seem to follow the convention of more nested headings = smaller. Will that be addressed in this ticket or will that be for Trevor's ticket this sprint?